<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hanoi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_a_b">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1.  A -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_b_a">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3.  B -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_c_a">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5.  C -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_a_c">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2.  A -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_b_c">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4.  B -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_c_b">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6.  C -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_disks_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Set disk amount (This resets the game)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_amount_num">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_amount_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum amount of moves to win:</string>
    </property>
   </widget>
   <widget class="QLabel" name="moves_done_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Moves done:</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_counter_num">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="moves_left_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Moves left to minimum amount of moves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="moves_left_num">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_disks">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>31</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
